### ISSUE TITLE
Move template to /usr/lib (or similar)

### ISSUE THREAD

**Issue by svenseeberg** [openSUSE/docserv#72]
There is lots of HTML and CSS which should not be located in /etc/. These files should be moved to /usr/lib or similar. This could be set up as one possible theme and packaged in another RPM.

**Comment by sknorr** [comment]
That is a possibility ... but given that even just for the official Docserv, we'd need 2 to 3 theme RPMs, I am not entirely convinced. I am also not sure whether packaging themes that way makes it easier to set up the server anew.

**Comment by tomschr** [comment]
I agree with Sven here. It makes packaging a bit more complicated as RPM complains about the files inside `/etc` (they need the `%config` macro). That makes things also cleaner in `/etc`.

Do we really need it *now*? That seems a [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) here. ;-)

Can't we add all these "themes" to `/usr/lib` or `/usr/share` *now*? We can always decide different later.
